tests/test_signer.py

```diff
@@ -35,7 +35,6 @@ async def send_transactions(signer, account, calls, nonce=None, max_fee=0):
     build_calls = []
     for call in calls:
         build_call = list(call)
-        build_call[0] = hex(build_call[0])
         build_calls.append(build_call)
```

Comment:
Since we don't need to hexify the first arg, we can remove all this and just pass `calls` to `get_raw_invoke` and `sign_transaction`

Also, let's remember to update `MockSigner` and `MockEthSigner` in Contracts before the next release since they use the same `build_calls` logic

Reply:
Fixed test.